example/cumulus-tf/lambdas.tf

```diff
@@ -139,3 +139,62 @@ resource "aws_lambda_function" "sns_s3_collections_test" {
     }
   }
 }
+
+resource "aws_secretsmanager_secret" "lzards_api_client_test_launchpad_passphrase" {
+  name_prefix = "${var.prefix}-api-launchpad-passphrase"
```

Comment:
This conflicts with https://github.com/nasa/cumulus/blob/master/tf-modules/archive/api.tf#L120
You maybe able to use the output from https://github.com/nasa/cumulus/blob/master/tf-modules/archive/outputs.tf#L53 instead of creating a new one.

Reply:
actually we should use `lzards_launchpad_passphrase`

Reply:
done. also removed conflict on name of secret.